api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -100,6 +100,9 @@ message Bootstrap {
   // Configuration for internal processing of stats.
   envoy.config.metrics.v2.StatsConfig stats_config = 13;
 
+  // Optional string matcher for allowing/disallowing the instantiation of stats by name.
+  envoy.config.metrics.v2.StatsFilter stats_filter = 16;
```

Comment:
WDYT about putting this inside StatsConfig? That seems like maybe a better place along w/ tags?

Reply:
That works for me. I see the advantage to keeping fewer top-level Bootstrap fields. (See 1f0cc3e)